lib/include/tree_sitter/api.h

```diff
@@ -605,25 +605,15 @@ TSStateId ts_node_next_parse_state(TSNode self);
 
 /**
  * Get the node's immediate parent.
- * Prefer [`ts_node_child_containing_descendant`] for
+ * Prefer [`ts_node_child_with_descendant`] for
  * iterating over the node's ancestors.
  */
 TSNode ts_node_parent(TSNode self);
 
-/**
- * @deprecated use [`ts_node_contains_descendant`] instead, this will be removed in 0.25
```

Comment:
Shouldn't that be `ts_node_child_with_descendant`?

Reply:
oops, yeah it should've been